.github/workflows/map_timeseries_tests.yml

```diff
@@ -0,0 +1,27 @@
+name: WHO timeseries script tests
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - '.github/workflows/cdc_who_gh_comparison_tests.yml'
+      - 'scripts/map_timeseries/*.py'
+      - 'scripts/map_timeseries/pyproject.toml'
+      - 'scripts/map_timeseries/poetry.lock'
+      - 'scripts/map_timeseries/docker-compose*'
+  pull_request:
+    paths:
+      - '.github/workflows/cdc_who_gh_comparison_tests.yml'
+      - 'scripts/map_timeseries/*.py'
+      - 'scripts/map_timeseries/pyproject.toml'
+      - 'scripts/map_timeseries/poetry.lock'
+      - 'scripts/map_timeseries/docker-compose*'
+
+
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run tests
+        run: cd scripts/map_timeseries && ./test_stack.sh
```

Comment:
This pattern will be the same for each script subdirectory, it would be awesome to have one .yml file for all scripts.